### PR TITLE
Update remaining modules to support Python 3.13

### PR DESCRIPTION
**Pull request by youknowone**
cc @lucas42

### CODE REVIEW

File: asynchat/asynchat/__init__.py [hunk 1]
```diff
@@ -48,12 +48,13 @@
...
 
-from warnings import _deprecated
-
-_DEPRECATION_MSG = ('The {name} module is deprecated and will be removed in '
-                    'Python {remove}. The recommended replacement is asyncio')
-_deprecated(__name__, _DEPRECATION_MSG, remove=(3, 12))
+import warnings
 
+# python-deadlib: Replace deprecation warning not to raise exception
+_DEPRECATION_MSG = (f'The {__name__} module is deprecated and be removed in '
```
Comment (lucas42) [inline asynchat/asynchat/__init__.py:54]:
```suggestion
_DEPRECATION_MSG = (f'The {__name__} module was removed in '
```

File: asyncore/asyncore/__init__.py [hunk 2]
```diff
@@ -57,9 +57,11 @@
      ENOTCONN, ESHUTDOWN, EISCONN, EBADF, ECONNABORTED, EPIPE, EAGAIN, \
      errorcode
 
-_DEPRECATION_MSG = ('The {name} module is deprecated and will be removed in '
-                    'Python {remove}. The recommended replacement is asyncio')
-warnings._deprecated(__name__, _DEPRECATION_MSG, remove=(3, 12))
+# python-deadlib: Replace deprecation warning not to raise exception
+_DEPRECATION_MSG = (f'The {__name__} module is deprecated and be removed in '
```
Comment (lucas42) [inline asyncore/asyncore/__init__.py:61]:
```suggestion
_DEPRECATION_MSG = (f'The {__name__} module was removed in '
```

File: mailcap/mailcap/__init__.py [hunk 3]
```diff
@@ -7,10 +7,12 @@
 __all__ = ["getcaps","findmatch"]
 
 
-_DEPRECATION_MSG = ('The {name} module is deprecated and will be removed in '
-                    'Python {remove}. See the mimetypes module for an '
-                    'alternative.')
-warnings._deprecated(__name__, _DEPRECATION_MSG, remove=(3, 13))
+# python-deadlib: Replace deprecation warning not to raise exception
+_DEPRECATION_MSG = ('The {__name__} module is deprecated and will be removed in '
```
Comment (lucas42) [inline mailcap/mailcap/__init__.py:11]:
```suggestion
_DEPRECATION_MSG = ('The {__name__} module was removed in '
```

File: smtpd/smtpd/__init__.py [hunk 4]
```diff
@@ -77,18 +77,19 @@
...
 from email._header_value_parser import get_addr_spec, get_angle_addr
 
 __all__ = [
     "SMTPChannel", "SMTPServer", "DebuggingServer", "PureProxy",
 ]
 
-_DEPRECATION_MSG = ('The {name} module is deprecated and unmaintained and will '
-                    'be removed in Python {remove}.  Please see aiosmtpd '
+_DEPRECATION_MSG = ('The {__name__} module is deprecated and '
+                    'be removed in Python 3.12.  Please see aiosmtpd '
```
Comment (lucas42) [inline smtpd/smtpd/__init__.py:88]:
```suggestion
_DEPRECATION_MSG = ('The {__name__} module was '
                    ' removed in Python 3.12.  Please see aiosmtpd '
```

**Review by lucas42 — COMMENTED** [review]
I think it's clearest to say "was removed" for each of these messages, rather than having different variations.

Note: not all the tests pass in 3.13 after this, so there may need to be some more tweaks before they fully support python 3.13.

File: smtpd/smtpd/__init__.py [hunk 5]
```diff
@@ -77,18 +77,19 @@
...
                     '(https://aiosmtpd.readthedocs.io/) for the recommended '
-                    'replacement.')
-_deprecated(__name__, _DEPRECATION_MSG, remove=(3, 12))
+                    'replacement. '
+                    'Please be aware that you are currently NOT using standard "{__name__}", but instead a separately installed "standard-{__name__}".'))
```
Comment (lucas42) [inline smtpd/smtpd/__init__.py:91]: Mis-matched brackets:
```suggestion
                    'Please be aware that you are currently NOT using standard "{__name__}", but instead a separately installed "standard-{__name__}".')
```

File: smtpd/smtpd/__init__.py [hunk 6]
```diff
@@ -77,18 +77,19 @@
...
 from email._header_value_parser import get_addr_spec, get_angle_addr
 
 __all__ = [
     "SMTPChannel", "SMTPServer", "DebuggingServer", "PureProxy",
 ]
 
-_DEPRECATION_MSG = ('The {name} module is deprecated and unmaintained and will '
-                    'be removed in Python {remove}.  Please see aiosmtpd '
+_DEPRECATION_MSG = ('The {__name__} module was '
+                    ' removed in Python 3.12.  Please see aiosmtpd '
```
Comment (youknowone) [inline smtpd/smtpd/__init__.py:88]:
```suggestion
                    'removed in Python 3.12.  Please see aiosmtpd '
```

**Comment by youknowone** [comment]
@lucas42 Thank you!

**Pull request merged**